docs/community/governance.md

```diff
@@ -0,0 +1,118 @@
+
+## Formal Side
+
+The GradleUp organization is a volunteer driven community
+operating in a public interest in a non-profit mode and
+providing no commercial services.
+As it is written in the Apache License v2:
+
+> Licensor provides the Work (and each Contributor provides its Contributions) on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied, including, without limitation, any warranties or conditions of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE. You are solely responsible for determining the appropriateness of using or redistributing the Work and assume any risks associated with Your exercise of permissions under this License.
+
+Gradle and the Gradlephant logo (the "Gradle Marks") are registered trademarks of [Gradle, Inc.](https://gradle.com/) and/or its subsidiaries.
+In the GradleUp organization, "Gradle" typically means "Gradle Build Tool" and does not reference Gradle Inc. and/or its subsidiaries.
+All references to the trademark represent a fair use,
+and do not imply endorsement.
+
+## Structure
+
+The project is led by a small team of passionate Gradle users and contributors.
+This [GradleUp Team](./team.md) is to be considered as a top governance and steering entity for the project.
+
+The project also includes maintainers and contributors.
+
+## Roles
+
+* **Contributor** - any individual who participates in the community and 
+  adds value to in one form or another,
+  including but not limited to code, documentation ,automation, advocacy or
+  community support.
+* **Team Member** -
+  A GradleUp contributor listed on the [GradleUp Team page](./team.md).
+  They participate in the project steering as whole.
+  New team members are added by the decision of the current team members.
+* **Org Admin** - A team member with administrative access to the GitHub
+  organization and/or other community resources.
+  Permissions and access are granted based on the decision of the current team members.
+* **Maintainer** - A GradleUp contributor who maintains a component hosted on GradleUp.
+  It includes roles _Triage_ or above.
+  Components may explicitly or implicitly introduce more Maintainer sub-roles.
+  
+## Decision Making
+
+The decisions are made by the [GradleUp Team](./team.md) based on consensus and common sense.
+We intend to make decision making fast and light-weight,
+with minimum overhead when a consensus exists.
+
+At the moment the decisions are made in a GradleUp Team private channel.
+Once [public communication channels](./participate.md) are established,
+it is expected that most of the discussions will happen there,
+and all contributors and end users will be able to contribute to the decisions.
+
+## Components Governance
+
+### Component Ownership
+
+_Maintainers_ have full control of technical decisions and processes within their components.
+Apart from the requirements documented in the [Hosting Guidelines](./hosting.md),
```

Comment:
👍 We should add a note in `hosting.md` that maven coordinates must be in the `com.gradleup` namespace

Reply:
I prefer to postpone it until there is a trademark grant